doc/releases/v03.02.adoc

```diff
@@ -0,0 +1,37 @@
+= https://github.com/OpenSimulationInterface/open-simulation-interface/releases/tag/v3.2.0[v3.2.0 - OSI "Editorial Eaton"]
+
+https://www.youtube.com/watch?v=SJUhlRoBL8M[Think p(OSI)tive!]
+
+== Purpose
+
+The work on the OSI project was until now mainly organized by the BMW AG. In order to enable as many interested parties as possible to contribute to the success of the Open Simulation Interface the project has been transferred to the https://www.asam.net/[ASAM e.V.] which has been made public in the https://www.asam.net/news-media/news/detail/news/bmw-transfers-open-simulation-interface-osi-to-asam/[ASAM press release].
+A new chapter in the development of OSI is now opened with the https://www.asam.net/conferences-events/detail/asam-osi-kick-off-workshop/[kickoff workshop] leading to hopefully many project proposals streamlining the efficient future development of the interface.
+Due to this we decided to focus on the improvement of processes and documentation with this release.
+
+== https://github.com/OpenSimulationInterface/open-simulation-interface/pulls?q=is%3Apr+is%3Aclosed+milestone%3Av3.2.0[Content/Changes]
+
+
+* Updated README according to K(eep) I(t) S(imply) S(tupid) principle
+* Added Issue and Pullrequest templates
+* Travis CI updated
+** Update to bionic 18.04
+** Added tests for building protobuf 2.6 and 3.0 in parallel
+** Bugfixed the display of doubled documentation
+* Fixed several typos in the comments
+* Added a centralized documentation in https://github.com/OpenSimulationInterface/osi-documentation[OSI-Documentation]
+** Added contributors guidelines
+** Added commenting style guidelines
+** Added uniform citation style according to APA
+** Updated installation guides for osi in Linux and Windows
+** Added a reference in sphinx based on the generated doxygen documentation 
+* Added rules to the comments in order to enforce osi message validation with the https://github.com/OpenSimulationInterface/osi-validation[OSI-Validator]
+* Added the rule definitions in the rules.yml
+* Modularized testing of commenting style into python unit tests
+* Defined a standardazied a file format for reading trace files (*.osi)
```

Comment:
```suggestion
* Defined a standardized a file format for reading trace files (*.osi)
```

Reply:
I did not want to make changes to already published release notes.

Reply:
decided in the CCB that we don't fix typos in already released release notes.